### PR TITLE
EZP-27906: As an Editor, I want to edit a Map Location field type

### PR DESCRIPTION
**Pull request by sunpietro**
| Question      | Answer
| ------------- | ---
| Requires    | https://github.com/ezsystems/repository-forms/pull/180
| Tickets       | https://jira.ez.no/browse/EZP-27906
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | ?
| Doc needed?   | ?
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Requires: https://github.com/ezsystems/ezplatform-admin-ui-assets/pull/3

**THINGS DONE:**
- [x] created base files for `ezgmaplocation` field,
- [ ] everything else

### CODE REVIEW

**Comment by webhdx** [comment]
This PR requires changes from https://github.com/ezsystems/repository-forms/pull/180

File: src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+     */
+    const preventFormSubmission = (event) => event.currentTarget.closest('form').addEventListener('submit', disableDefaultAction, false);
+
+    /**
+     * Enables form submission
+     *
+     * @function enableFormSubmission
+     * @param {Event} event
+     */
+
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js]: empty line

File: src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+    const enableFormSubmission = (event) => event.currentTarget.closest('form').removeEventListener('submit', disableDefaultAction);
+    /**
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/fieldType/ezgmaplocation.js]: missing empty line

**Comment by sunpietro** [comment]
ping @dew326

**Pull request merged**